test/staging/ArrayBuffer/resizable/array-fill-parameter-conversion-resizes.js

```diff
@@ -0,0 +1,109 @@
+// Copyright 2021 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-arraybuffer-length
+description: >
+  Automatically ported from ArrayFillParameterConversionResizes test
+  in V8's mjsunit test typedarray-resizablearraybuffer.js
+includes: [compareArray.js]
+features: [resizable-arraybuffer]
+flags: [onlyStrict]
+---*/
+
+class MyUint8Array extends Uint8Array {
```

Comment:
I'm not a fan of this duplication either... but my aim was to get a set of correct tests that fulfill the conditions to land in staging, and not worry too much about concerns like this. I'd think this would be made obsolete when we move the resizable-arraybuffer tests into the main test262 suite with https://github.com/tc39/test262/pull/3493 anyway.

An alternative could have been to keep these things in some sort of "common" file as they were in the original V8 tests from which they were converted. There isn't currently a mechanism to do that; there are "common" helper files in the `harness/` directory but they are not subject to the relaxed rules for landing in staging. I think if this comes up more often we should consider having some sort of `staging-common` directory that the frontmatter `includes:` can reference, but for this case I think I'd personally prefer to focus on https://github.com/tc39/test262/pull/3493.

Reply:
Ok, makes sense to worry about it later.